### ISSUE TITLE
Panic error when i try to send documents to index

### ISSUE THREAD

**Issue by gmourier** [meilisearch/MeiliSearch#1453]
**Describe the bug**
I've got a panic error when I try to index the movies dataset onto the RC v0.21 instance hosted on the MeilISearch cloud but it also happened with a 3-4 document set.

Weird since @eskombro seems to have managed to index documents.

I can give you access to my instance if it helps! 

**To Reproduce**
1. Try to add the movies dataset

**Expected behavior**
I was expecting to get an updateId

**Desktop (please complete the following information):**
 - Postman

**Additional context**
MeiliSearch Response 
```
{
    "message": "Internal error: panic",
    "errorCode": "internal",
    "errorType": "internal_error",
    "errorLink": "https://docs.meilisearch.com/errors#internal"
}
```

**Comment by curquiza** [comment]
Can you send us the logs? Because I don't have the issue locally

**Comment by curquiza** [comment]
We found out the bug, it's related to #1455

**Comment by gmourier** [comment]
Indeed it was blazing fast! 😄

**Comment by curquiza** [comment]
Fixed by #1456
